src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import { Person } from './types';
+import { getPeople } from '../api';
+import { PeopleList } from './PeopleList';
+import { Loader } from './Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [persons, setPersons] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
you can initially set loading to true - since the component will be already fetching you can still unset it in `.finally`